solution/ui/regulations/js/src/components/SupplementalContentList.vue

```diff
@@ -47,7 +47,7 @@
             >
             </supplemental-content-object>
             <collapse-button
-                v-if="showMoreNeeded && contentCount > 10"
+                v-if="showMoreNeeded && contentCount > limit"
```

Comment:
It shouldn't be changed here because it is working as designed.  See Subregulatory Guidance > Frequently Asked Questions (FAQs) here:

https://regulations-pilot.cms.gov/42/435/Subpart-B/2019-05-06/

If there are more than `limit` items (in this case five) but fewer than 10, then this button should be hidden.  This was discussed in a long validation thread you can read here:

https://fearless.slack.com/archives/C02DL6UU076/p1636723469000900

The same location on your branch [here](https://dd7ssdpi8b.execute-api.us-east-1.amazonaws.com/dev546/42/435/Subpart-B/2019-05-06/) shows two "show less" buttons (one after the fifth item and one at the end of the list) when there should only be one (after the fifth item).